kolibri/core/assets/src/views/AppBar.vue

```diff
@@ -219,6 +220,15 @@
           }, this.pollingInterval);
         }
       },
+      setPollingInterval(status) {
+        if (status === SyncStatus.QUEUED) {
+          // check more frequently for updates if the user is waiting to sync,
+          // so that the sync isn't missed
+          this.pollingInterval = 1000;
```

Comment:
Occurs to me looking at this now, that technically `this.pollingInterval` doesn't need to be reactive, so we could just set it in `created` rather than in data. Doesn't need to be changed though!